hypha/apply/funds/models/utils.py

```diff
@@ -116,7 +116,8 @@ def render_landing_page(self, request, form_submission=None, *args, **kwargs):
     content_panels = AbstractStreamForm.content_panels + [
         FieldPanel('workflow_name'),
         InlinePanel('forms', label=_('Forms')),
-        InlinePanel('review_forms', label=_('Review Forms')),
+        InlinePanel('review_forms', label=_('Internal Review Forms')),
+        InlinePanel('external_review_forms', label=_('External Review Forms')),
```

Comment:
If external review form is always gonna be one..ideal way would be to just add a relational field to select a review form..instead of adding InlinePanel which provides the possibility of adding multiple of them.

Although, in terms of UI, this way of adding external review form goes along with what have for adding other forms. And we also have the validation which raises error if it is more than one. But would be good if we could limit the addition of forms to only one. 

@sandeepsajan0 can you check if it is possible to set max number of items that can be added for a InlinePanel.

Reply:
Yes, it is possible. It has a `max_num` property that can disable the Add button when the limit is reached. I'll add it for External Review form.